### ISSUE TITLE
git submodule instructions

### ISSUE THREAD

**Issue by FrankReh** [timothee-haudebourg/json-ld#43]
The tests/README.md says to run *git submodules update* but I don't find a git-submodules tool. There is however a git-submodule tool but that seems to require two steps, calling with init and then with update. Then the tests can be run.
```sh
git submodule init
git submodule update
cargo test
```

**Comment by timothee-haudebourg** [comment]
That's right, I even do the `init` in the CI workflow. You can do a PR if you want, or I'll fix that myself when I have some time.
